mtags/src/main/scala-2/scala/meta/internal/metals/MultilineStringFormattingProvider.scala

```diff
@@ -251,6 +325,12 @@ object MultilineStringFormattingProvider {
     List(textEditPrecedentLine, textEditcurrentLine)
   }
 
+  private def repaceWithSixQuotes(pos: Position): List[TextEdit] = {
+    val pos1 = new Position(pos.getLine, pos.getCharacter - 3)
```

Comment:
Will it work if someone adds a quote inside existing quotes? `"add new quote here"`

Reply:
This can only be trigger if there are already 3 consecutive quotes and the fourth character being typed is a quote, and the file cannot be tokenized.

Reply:
Actually I just found one case where it will introduce a bug.

If there is already line = """ whatever 
and then the user will try to write 3 quotes. 
Also, I wonder how quotes are being written in other editors. For vscode it's will always write two quotes by two quotes, but maybe 3 is enough ?

Reply:
if we have an existing string say:
`val a = ""`
and we add quotes before, we will get:
`val a = """"`
Maybe we should also create 6 quotes there?

Reply:
It's clearly not a normal usage of this feature (adding quotes in the beginning ). I don't think we should take this usage into account.
I don't know for the "find the end and fill until 6": it means it will only cover the 3 quotes case since you are not supposed to have 5 quotes except if you deleted two quotes

Reply:
4 seems like a good number for me, since 3 could be a typing error.

Reply:
Ok, let's leave it this way. I might have been overthinking this.